checkout_session.go

```diff
@@ -157,6 +157,7 @@ const (
 	CheckoutSessionCustomerDetailsTaxIDTypeGBVAT    CheckoutSessionCustomerDetailsTaxIDType = "gb_vat"
 	CheckoutSessionCustomerDetailsTaxIDTypeGEVAT    CheckoutSessionCustomerDetailsTaxIDType = "ge_vat"
 	CheckoutSessionCustomerDetailsTaxIDTypeHKBR     CheckoutSessionCustomerDetailsTaxIDType = "hk_br"
+	CheckoutSessionCustomerDetailsTaxIDTypeHROib    CheckoutSessionCustomerDetailsTaxIDType = "hr_oib"
```

Comment:
We need to add this to "acronyms"

Reply:
Hmm... why did all the checks pass... I recollect some validation checks failing for this before